tests/models_tests/models_test.go

```diff
@@ -195,25 +197,32 @@ func TestRetrieveInfoBySourceIdAndConceptTypesWrongType(t *testing.T) {
 	}
 }
 
-func TestRetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIdsNoResults(t *testing.T) {
+func TestRetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIdsAndCohortPairsNoResults(t *testing.T) {
 	setUp(t)
-	stats, _ := conceptModel.RetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIds(testSourceId,
+	// empty:
+	filterCohortPairs := make([][]int, 0)
+	stats, _ := conceptModel.RetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIdsAndCohortPairs(testSourceId,
 		smallestCohort.Id,
-		allConceptIds, allConceptIds[0])
+		allConceptIds, filterCohortPairs, allConceptIds[0])
 	// none of the subjects has a value in all the concepts, so we expect len==0 here:
 	if len(stats) != 0 {
 		t.Errorf("Expected no results, found %d", len(stats))
 	}
 }
 
-func TestRetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIdsWithResults(t *testing.T) {
+func TestRetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIdsAndCohortPairsWithResults(t *testing.T) {
 	setUp(t)
 	filterIds := make([]int64, 1)
 	filterIds[0] = hareConceptId
+	filterCohortPairs := make([][]int, 1)
```

Comment:
just wondering why we don't do something like 
` filterCohortParis := [][]int{
		{largestCohort.Id},
		{largestCohort.Id},
	}`

Reply:
nice! I'll use that. I only recently found that notation when looking at another part of your code 👍